fbpic/lpa_utils/boosted_frame.py

```diff
@@ -219,28 +219,6 @@ def wavenumber( self, lab_frame_vars ):
 
         return( boosted_frame_vars )
 
```

Comment:
The functions below are not needed anymore because of the new way in which the boosted frame is performed.